jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -394,7 +414,11 @@ MethodType accessModeType(Class<?> receiver, Class<?> type, Class<?>... args) {
 			int numAccessModes = accessModes.length;
 	
 			/* The first argument in AccessType.GET MethodType is the receiver class. */
+/*[IF Java16]*/
+			Class<?> receiverActual = accessModeTypeUncached(AccessMode.GET.at).parameterType(0);
+/*[ELSE]*/
 			Class<?> receiverActual = accessModeTypeUncached(AccessMode.GET).parameterType(0);
+/*[ENDIF] Java16 */
```

Comment:
Considering the difference is the argument, I would suggest 
```suggestion
Class<?> receiverActual = accessModeTypeUncached(
/*[IF Java16]*/
  AccessMode.GET.at
/*[ELSE]*/
AccessMode.GET
/*[ENDIF] Java16 */
).parameterType(0);
```